outlookmail/client.bal

```diff
@@ -0,0 +1,454 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/io;
+
+# Ballerina Client for microsoft outlook mail operations
+@display {
+    label: "Microsoft Outlook.mail Client",
+    iconPath: "MSOutlookMailLogo.svg"
+}
+public client class Client {
+    http:Client httpClient;
+    Configuration config;
+
+    # Client Initialization
+    #
+    # + config - Configuration for client connector
+    # + return - If success returns null otherwise returns the relevant error  
+    public isolated function init(Configuration config) returns error? {
+        self.httpClient = check getOutlookClient(config, BASE_URL);
+        self.config = config;
+    }
+
+    # Gets the messages in the signed-in user's mailbox (including the Deleted Items and Clutter folders)
+    #
+    # + folderId - The ID of the specific folder in the user's mailbox or the name of well-known folders (inbox, 
+    # sentitems etc.)
+    # + optionalUriParameters - The optional query parameter string
+    # (https://docs.microsoft.com/en-us/graph/api/user-list-messages?view=graph-rest-1.0&tabs=http#optional-query-parameters)
+    # + return - If success returns a ballerina stream of message records otherwise the relevant error
+    @display {label: "List Messages"}
+    isolated remote function listMessages(@display {label: "Folder ID"} string? folderId = (), 
+                                        @display {label: "Optional Query Parameters"} string? optionalUriParameters = ()) 
+                                        returns @tainted error|stream<Message, error?> {
```

Comment:
Do we need taint check in beta?

> The Taint analyzer is disabled and the --taint-check option is removed. The annotations `@tainted` and `@untainted` were left intact as placeholders for backward compatibility. Any script that used to pass the --taint-check option to the bal run or bal build commands will fail.
Ref: https://ballerina.io/downloads/swan-lake-release-notes/swan-lake-beta1/#breaking-changes